frontend/src/assets/locales/en/otherUses.json

```diff
@@ -22,14 +22,17 @@
   "addRow": "Add row",
   "rows": "rows",
   "otherUsesColLabels": {
-    "complianceReportId": "Compliance report ID",
-    "fuelType": "Fuel type",
-    "fuelCategory": "Fuel category",
-    "quantitySupplied": "Quantity supplied",
-    "addressForService": "Address for service",
-    "legalName": "Legal name",
+    "complianceReportId": "Compliance Report ID",
```

Comment:
We don't use title case normally, was this part of the requirements?

Reply:
Thank you!